src/mermaid.spec.js

```diff
@@ -4,6 +4,9 @@ import flowDb from './diagrams/flowchart/flowDb';
 import flowParser from './diagrams/flowchart/parser/flow';
 import flowRenderer from './diagrams/flowchart/flowRenderer';
 import Diagram from './Diagram';
+import { addDiagrams } from './diagram-api/diagram-orchestration';
+
+addDiagrams();
```

Comment:
Should we move diagram tests out from `mermaid.spec.js` ?

Reply:
There are some tests related to flowcharts in there, those would be more suited in flowcharts! 👍